Makefile.work

```diff
@@ -31,6 +31,9 @@
 #  *                 Default: yes
 #  *                 Values:  yes, no
 #  * KERNEL_PROCURE_METHOD: Specifying method of obtaining kernel Debian package: download or build
+#  * TELEMETRY_READWRITE: Enable write/config operations via the gNMI interface.
```

Comment:
change to SONIC_TELEMETRY_READWRITE

Reply:
Okay, I have added the SONIC_ prefix to the telemetry readonly flag.